chore: added tech for Ukraine (#6653)
vuestorefront/vue-storefront: README.md

  <img src="https://user-images.githubusercontent.com/1626923/137092657-fb398d20-b592-4661-a1f9-4135db0b61d5.png" alt="Vue Storefront" height="80px" />
</div>

---------

> # #TechForUkraine

Reviewer: ```suggestion
# TechForUkraine
```

<table>
  <tr>
    <td style="width:40%;">
       <img src="https://user-images.githubusercontent.com/1626923/155853691-d6d0a541-d3b9-40bf-b8f5-2d38303e9e49.png" />
    </td>
    <td>
      <h2><strong>Ongoing tensions on Ukrainian territory close the space for civil society.</strong></h2>
      <h3>How can you support Ukrainian civil society?</h3>

>       All the help is valid, and if you are not able to help locally, by giving shelter to a fellow Ukraine, there are some ways that you can help also:

Reviewer: ```suggestion
      All help is needed. If you are unable to help locally, by sheltering a fellow Ukrainian, you can also:
```
